includes/wikia/services/tests/SassInlineImageFilterTest.php

```diff
@@ -1,18 +1,18 @@
 <?php
 
-class SassBase64FilterTest extends WikiaBaseTest
+class SassInlineImageFilterTest extends WikiaBaseTest
 {
 	/**
 	 * @dataProvider getProcessData
 	 */
 	public function testProcess($input, $output) {
-		$filter = $this->getMockBuilder( 'Wikia\Sass\Filter\Base64Filter' )
+		$filter = $this->getMockBuilder( 'Wikia\Sass\Filter\InlineImageFilter' )
 			->disableOriginalConstructor()
-			->setMethods( [ 'encodeFile' ] )
+			->setMethods( [ 'inlineFile' ] )
 			->getMock();
 
 		$filter
-			->method( 'encodeFile' )
+			->method( 'inlineFile' )
```

Comment:
Do you think adding a test case with `/* inline */` in `getProcessData()` and cleaning it at the time of retiring `/* base64 */` would be too big unnecessary cost? :money_with_wings: 

Also, unit tests don't cover [new condition](https://github.com/Wikia/app/blob/CON-svg/includes/wikia/services/sass/Filter/InlineImageFilter.class.php#L78) connected to `$enableBase64` which is being set depending on a file extension - this could be added with the mentioned additional test case in the data provider :japanese_ogre: